modules/ROOT/pages/apicat-create-descriptor-file-manually.adoc

```diff
@@ -0,0 +1,226 @@
+= Creating or Updating Descriptor Files Manually
+
+The 'create-descriptor` and `update-descriptor` commands identify the APIs to catalog and output the essential information needed to catalog them. If you want to add more descriptor information, the most efficient approach is to first create or update the descriptor file using the CLI, and then manually update the file with information that cannot be automatically. Use the following information about the descriptor YAML schema to decide what to add. 
```

Comment:
```suggestion
The 'create-descriptor` and `update-descriptor` commands identify the APIs to catalog and output the essential information needed to catalog them. If you want to add more descriptor information, create or update the descriptor file using the CLI, and then manually update the file with information that cannot be automatically. Use the following information about the descriptor YAML schema to decide what to add. 
```

Reply:
This phrase is in lieu of "MuleSoft recommends", which I recall our style prohibits. Not having some caveat is misleading, because it is possible to just create the file manually, and some people may want to do that.

Reply:
We are also not supposed to say "You can", but perhaps "you might want to..."?